src/fast_graph_builder.rs

```diff
@@ -227,7 +227,7 @@ impl FastGraphBuilder {
                 return edge_id;
             }
         }
-        panic!["could not find out-edge id"]
+        panic!("could not find out-edge id")
```

Comment:
I also noticed this. I guess technically the panic macro allows any kind of delimiter, but I've only ever seen `()` used

Reply:
Oh wow, thanks for figuring this out, I could not make sense of the error message :+1: